dist/isoCurrency.js

```diff
@@ -894,8 +894,16 @@ angular.module('isoCurrency', ['isoCurrency.common'])
 			if (!currency) {
 				return amount;			
 			}
+
 			var fractionSize = (fraction === void 0) ? currency.fraction : fraction;
-			return $filter('currency')(amount, currency.symbol, fractionSize);
+	        var currencyFilter = $filter('currency');         
+
+	        if(amount < 0){
```

Comment:
Yes, leave that as a comment then.  The thing is, it's a design decision, not a technical implementation.  We don't have a design doc to point to and say "this is what it is".  It's up to your gut (which you clearly have an opinion on here).  That said, 6 months from now when 1 Guest tells us "hey it should be parenths" or whatever, that comment will remind me, or Ross, why that decision was made in the first place.

Startup lesson - Remaking and unmaking decisions that have already been made is one of the most expensive things you can do in a startup.